### PR TITLE
2839 fix unfpa dashboards missing fiji

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
[#2839](https://github.com/beyondessential/tupaia-backlog/issues/2839)

### Changes:
apply logic from [this comment](https://github.com/beyondessential/tupaia-backlog/issues/2839#issuecomment-851087096)

### CODE REVIEW

File: packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const filterFacility = async (models, filterCriteria, analytics) => {
   const facilities = await models.facility.find({
     type: {
       comparator: filterCriteria.comparator,
-      comparisonValue: '1',
+      comparisonValue: '1%',
```
Comment (nick-beyondessential) [inline packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js:19]: Should work as long as we don't add more than 9 facility types. :thinking:

Comment (billli0) [reply packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js:19]: oh yeah @nick-beyondessential that's a good point. Not sure if that works with Tupaia but I will try `"type" similar to '(1|1.%)'`, which should handle more than 9 facility types.

Comment (nick-beyondessential) [reply packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js:19]: I wouldn't stress too much about it - not sure how likely it is.

Comment (billli0) [reply packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js:19]: Will keep that in mind.

**Review by nick-beyondessential — APPROVED** [review]
LGTM

**Pull request merged**